client/store/pages/dataExport/hooks/index.ts

```diff
@@ -0,0 +1,48 @@
+import { Country, RegionCode } from '@meta/area'
+import { Objects } from '@core/utils'
+
+// import { useCountryIso, useAssessmentType, useCountries, useCountriesPanEuropean } from '@webapp/store/app'
+import { useCountryIso } from '@client/hooks'
+import { useCountries } from '@client/store/assessment'
+
+// import { useHomeCountriesFilter } from '@webapp/store/page/home'
+
+import { DataExportActions, DataExportSelection } from '@client/store/pages/dataExport'
+import { useAppDispatch, useAppSelector } from '@client/store'
+
+export const useDataExportCountries = (): Array<Country> => {
+  const dispatch = useAppDispatch()
+  // const assessmentType = useAssessmentType()
+  const countries = useAppSelector((state) => state.pages?.dataExport?.countries)
+
+  // if (assessmentType === AssessmentType.panEuropean) {
+  //   return useCountriesPanEuropean()
+  // }
+
+  const countryIso = useCountryIso()
+  const countriesAll = useCountries()
+  // const countriesFilter = useHomeCountriesFilter()
```

Comment:
add this hook , that returns an empty object, and uncomment code below where it's used